constellation-app/constellation
# Fix incorrect warning when opening file

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Fixed bug that would cause a warning to be shown, telling users that the file they were opening in word cloud would be overridden.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Bug fix, minimizing confusion.

### Possible Drawbacks


None.

### Verification Process



1. Open Consty
2. Open world cloud view
3. Open file in "background File" parameter
4. Observe that override warning is no longer shown when opening file.

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2111
